proc.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Prometheus Authors
+// Copyright 2019 The Prometheus Authors
```

Comment:
I think this copyright date should remain 2018 since the file already existed before this change.

Reply:
Okay, I'll change it now